external.bzl

```diff
@@ -1,11 +1,24 @@
 load("@bazel_gazelle//:deps.bzl", "go_repository")
 
 def go_dependencies():
+    go_repository(
+        name = "cc_mvdan_gofumpt",
+        importpath = "mvdan.cc/gofumpt",
+        sum = "h1:bi/1aS/5W00E2ny5q65w9SnKpWEF/UIOqDYBILpo9rA=",
+        version = "v0.1.1",
+    )
+    go_repository(
+        name = "cc_mvdan_xurls_v2",
+        importpath = "mvdan.cc/xurls/v2",
+        sum = "h1:NSZPykBXJFCetGZykLAxaL6SIpvbVy/UFEniIfHAa8A=",
+        version = "v2.2.0",
+    )
+
```

Comment:
please manually remove this blank line and all other blank lines in this file added by gazelle (see: https://github.com/bazelbuild/bazel-gazelle/issues/1088)